benchmarks/config/base/vector-to-kernel.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+  "prepacked_targets_vector_kernel": {
+    "gemm_fp32_mlir": {
```

Comment:
Move these into the block `prepacked_targets` in `base.json` and name them:
* `gemm_fp32_vector`
* `mlp_fp32_vector`

Reply:
Same as above. Moved into `prepacked_targets` and removed the `base\vector-to-kernel.json`